### ISSUE TITLE
Support bulk delete

### ISSUE THREAD

**Issue by mtlynch** [mtlynch/picoshare#327]
It's tedious to delete multiple files since you have to go through the delete flow for each one-by-one.

We should add some mechanism that allows you to delete multiple items quickly.

**Comment by xHyperElectric** [comment]
I agree, it would be awesome if on the files page there was a check box beside every file so you could delete many files at once, rather than click edit, then delete on each file
